py/desiutil/test/test_plots.py

```diff
@@ -8,13 +8,14 @@
 import unittest
 import os
 import numpy as np
-
+import sys
 # Set non-interactive backend for Travis
 import matplotlib
 matplotlib.use('agg')
 import matplotlib.pyplot as plt
 
 from ..plots import plot_slices
+#from ..plots import plot_sky
```

Comment:
You've commented out this import, but that means that `test_plot_sky()` will be broken on systems that are *not* Travis.